### DISCUSSION TITLE
Why would you do that (review)

### DISCUSSION THREAD

**Discussion by thoughtsunificator** [kovidgoyal/kitty#8095] [General]
Moving from xterm.

I'm mostly spending my time unconfigurating the  (wrong) defaults, such as (mostly everything):

- confirm_os_window_close 
- enable_audio_bell
- remember_window_size
- copy_on_select
- ctrl+tab
- Sometimes, links fails to open when you click on them (while they do open using ls) or by using the vimium kind of looking interface.
- The kitten ssh thing

There's also the fact that it takes a hell lot of time to launch kitty compared to xterm on my computer.

Not everyone is using a [name every inefficient window manager], I'm using i3, I could be using dwm, the point is, remembering window positions and tabs features are useless (at the very least dont default to them), there are multiplexer (for the tabs thing), and you usually don't want to handle your window positioning on a second basis.

I had hope this terminal was aiming for the minimalism of st while actually being usable, but I'm wondering... Actually this project reminds me of Hyper...

With all that being said, there ARE good features such as the repositioning of the buffer which allows me to full screen and un full screen without having to scroll back every damn time. 

Thanks

**Comment by kovidgoyal** [comment]
Why do you labor under the delusion that what you think are correct
defaults for you are correct for everyone. And use kitty
--single-instance then it will startup 10 times faster than xterm.

**Reply by thoughtsunificator** [reply]
It makes no sense to me that you would force tabs by default and then point out that it is my opinion that there should not be tabs by default. Being non-opinionated means the choice is left out to the end-user. The more choices you take the less choice we have to take, or, at the very least, we have to reconfigure like I do. These kinds of features have nothing to do with Terminals so basically, we're adding more workload on features that are already worked on, above in the stack which inevitably means that core features will be getting less attention than they deserve.

Thank you for the tips by the way and all the time spent on documentation, actually, I like a lot of things I see about this project, but yeah, I mean, the defaults are horrendous, and it should be addressed...

**Reply by kovidgoyal** [reply]
I have no idea what force tabs means. If you dont like tabs dont use them, no one is forcing you to use them. Since you mentioned st I am guessing you belong to the school of programmers that is incompetent enough to believe that the only way to have good software is to make it do as little as possible. The existence of tabs does not in any way prevent kitty from having more features and better performance and greater robustness than other terminal emulators. Once again, just because *you* dont like tabs is not a sufficient reason to try to deny everyone else from using them. 

And for the future, if you hope to engage productively with a project never show up and make asinine statements like "all your defaults are wrong". Good luck.
